stable/rabbitmq/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: rabbitmq
-version: 0.5.2
```

Comment:
It would be a breaking change if someone just update the docker image in the chart but not if they run `helm upgrade`